deploy/kubernetes/webhook-example/create-cert.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
```

Comment:
we might need to provide the source reference of this file. same for patch-ca-bundle.sh

Reply:
How can we properly credit the source for this file which is https://github.com/banzaicloud/admission-webhook-example/blob/blog/deployment/webhook-create-signed-cert.sh?

Reply:
Which was originally from https://github.com/istio/istio/blob/release-0.7/install/kubernetes/webhook-create-signed-cert.sh? They are under Apache 2.0 licenses.